source/_includes/_adminpanel/domains-ce.rst

```diff
@@ -249,12 +252,93 @@ CERTIFICATE`. In the dialog, you can choose to use:
   notification will be shown (:bdg-success:`The certificates have been
   saved`). To complete the procedure: if you are on a Single-Node,
   restart it otherwise you need to restart the node on which the
-  **Proxy** is installed;
-
+  **Proxy** is installed.
 
 You can :red:`REMOVE` or :blue:`DOWNLOAD` the certificates
 by clicking the appropriate button above the certificates themselves.
-                  
+
+.. _le-procedure:
+
+Procedure to install a Let's Encrypt certificate
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. card:: Let's Encrypt Requirements
+
+   Before attempting to ask for a Let's Encrypt certificate, make sure
+   that:
+
+   #. **Public Service Protocol** and **Public Service Host Name** are
+      correctly set in the |adminui|'s :menuselection:`Domain -->
+      General Settings`
+
+   #. There is a Virtual Host correctly configured for the domain you
+      want the certificate
+
+   #. **A**, **AAAA**, and **NXDOMAIN** record in the domain's DNS
+      configuration
+
+   #. The domain has a valid |fqdn| that can be resolved from anywhere
+      (i.e., the domain must be publicly accessible)
+
+   #. The domain (i.e., the Node which installs the Proxy) is
+      reachable from the Internet. In case the proxy is behind a
+      firewall or other routing devices and can not be reached
+      directly, add some forwarding rule on the firewall or device.
```

Comment:
What do you mean here with "or device"? the client?
LE need to connect to the http/80 port